test/unit/api/multi-profile.test.ts

```diff
@@ -0,0 +1,110 @@
+import { describe, expect, it } from "bun:test";
```

Comment:
Let's move it to examples/typescript-us-core to make tests self-contained. It can be checked in the same way as we already do it for other examples like r4.

In Makefile, you can find the target to run tests. Also, on the CI.